pkg/operator/workloadcontroller/workload_controller.go

```diff
@@ -115,7 +115,9 @@ func (c ServiceCatalogAPIServerOperator) sync() error {
 		}
 		return nil
 	default:
-		c.eventRecorder.Warningf("ManagementStateUnknown", "Unrecognized operator management state %q", operatorConfig.Spec.ManagementState)
+		if len(operatorConfig.Spec.ManagementState) > 0 {
+			c.eventRecorder.Warningf("ManagementStateUnknown", "Unrecognized operator management state %q", operatorConfig.Spec.ManagementState)
+		}
```

Comment:
why isn't an empty management state also considered to be an unrecognized management state and reported as such?  seems like it's still an error condition in some fashion.

Reply:
Perhaps it should be.  This was my understanding in conversations with @derekwaynecarr, but perhaps that was just step along the path to getting Service Catalog into the CVO payload with it disabled.  We are now setting it to an initial state of removed.  I'll revisit this along with the rework above.